app/services/state_service.rb

```diff
@@ -39,17 +39,17 @@ def object_state
     STATES[:unlock_inactive]
   end
 
+  private
+
+  attr_reader :druid, :version
+
   ##
   # Ported over logic from app/helpers/dor_object_helper.rb#LN133
   # @return [Boolean]
   def accessioned?
```

Comment:
Should this be removed after (or as part of) this PR?

Reply:
Yes, it will be removed once StateService is refactored. (Forthcoming PR.)